src/Rule.php

```diff
@@ -1379,7 +1379,7 @@ public function dropdownActions($options = [])
     /**
      * Get a criteria description by his ID
      *
-     * @param integer $ID the criteria's ID
+     * @param int|string $ID the criteria's ID
```

Comment:
I think criteria keys are always strings.

```suggestion
     * @param string $ID the criteria's ID
```